Description: Fix setup.cfg to use tool:pytest instead of just pytest
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2021-10-27

--- python-pecan-1.3.3.orig/setup.cfg
+++ python-pecan-1.3.3/setup.cfg
@@ -5,5 +5,5 @@ nocapture=1
 cover-package=pecan
 cover-erase=1
 
-[pytest]
+[tool:pytest]
 norecursedirs = +package+ config_fixtures docs .git *.egg .tox
